datasets/pl/wanted/pl_wanted.yml

```diff
@@ -0,0 +1,76 @@
+name: pl_wanted
+title: "Poland Wanted Persons"
+entry_point: crawler.py
+prefix: pl-wanted
+coverage:
+  frequency: daily
+  start: 2025-02-04
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  Wanted and suspect persons list of the Police of Poland
+description: |
+  Dataset with name, crime and status (suspect or wanted) published
+  by the Police of Poland.
+
+  Additional characteristic features of the wanted individuals are available at the source URL.
+
+publisher:
+  name: Policja
+  name_en: Police of Poland
+  description: |
+    Policja is the Polish national civilian police force. It is a primary law enforcement agency in the country,
+    tasked with providing public safety and order, investigating and preventing crime.
+  country: pl
+  official: true
+  url: https://www.policja.pl
+
+data:
+  url: https://poszukiwani.policja.gov.pl/pos/form/5,Poszukiwani.html
+  format: HTML
+  lang: pol
+
+dates:
+  formats: ["%Y-%m-%d"]
+
+assertions:
+  min:
+    schema_entities:
+      Person: 40000
+  max:
+    schema_entities:
+      Person: 60000
+
+lookups:
+  type.gender:
+    options:
+      - match: mężczyzna
+        value: male
+      - match: kobieta
+        value: female
+  type.country:
+    options:
+      - match:
+          - NIEUSTALONE  # unknown
+          - BEZPAŃSTWOWIEC  # stateless
+        value: null
+      - match: "JUGOSŁAWIA (SERBIA I CZARNOGÓRA)"  # TODO: What the hell should we map this to?
+        value: yucs
+      - match: "JUGOSŁAWIA"  # TODO: What the hell should we map this to?
+        value: yucs
+      - match: "NIEMCY JUGOSŁAWIA (SERBIA I CZARNOGÓRA)"  # TODO: What the hell should we map this to?
```

Comment:
we got you, fam https://www.opensanctions.org/reference/#:~:text=Serbia%20and%20Montenegro

Reply:
With the new logic, it'll just use JUGOSŁAWIA and match that to yucs.

Serbia and Montenegro doesn't exist anymore either, so not sure that's more useful. WDYT?

Reply:
we just try and represent the data as it is at the source, so we do normally map to these territories

Reply:
I guess I'm not quite sure what the intention of the source is here.

Anyway "JUGOSŁAWIA (SERBIA I CZARNOGÓRA)" now maps to yucs (because we throw away anything in parens). Are we okay with that or should I modify to deal with this and map to both yucs and csxx?

Reply:
I guess they mean the person was from the part of yugoslavia which became "serbia and montenegro"? But I haven't confirmed that the bit in parents is always "another form" of the same thing.

do whatever's easier between verifying that the bit in parentheses always means "another form of the bit before the parens", or just playing it safe and always including both.

We try not to throw information away, and maybe it's ok here, but my gut says it's not that hard to include both.